src/Domain/Infra/Doctrine/MappingConfig.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MsgPhp\Domain\Infra\Doctrine;
+
+/**
+ * @author Pascal Wacker <hello@pascalwacker.ch>
+ */
+final class MappingConfig implements MappingConfigInterface
+{
+    private $mappingFiles;
+    private $configDir;
```

Comment:
should really be called `$mappingDir`, so it's clear  this directory contains only mappings and not "any" config.